packages/popup/src/pages/CurrentSeed/index.js

```diff
@@ -3,21 +3,27 @@ import Input from '@cfxjs/component-input'
 import Button from '@cfxjs/component-button'
 import {Selected} from '@cfxjs/component-icons'
 import {CompWithLabel} from '../../components'
-import create from '../../hooks/zustand.js'
+import create from '../../hooks/zustand'
 
 const useStore = create(
   (set, get) => ({
     // value
     hdGroup: [],
     creatingAccount: false,
-    accountCreatationError: null,
+    accountCreationError: null,
     accountName: '',
     selectedGroup: null,
 
     // hook
     groupAfterSet: ({groupData}) => {
-      set({hdGroup: groupData.filter(({vault: {type}}) => type === 'hd')})
-      if (!get().selectedGroup) set({selectedGroup: get().group.groupData[0]})
+      const hdGroup = groupData.filter(g => g?.vault?.type === 'hd')
+      set({hdGroup})
+      if (!get().selectedGroup) set({selectedGroup: hdGroup[0]})
+      if (!get().accountName && hdGroup[0]) {
```

Comment:
there's a case that user want to create an account right after importing a seed phrase

the background will busy "discovering" additional accounts of this group. so there might
be a chance that the group's accounts get updated when user still on this page

Reply:
use `index` to get the selectedGroup